src/hooks/usePermit2.ts

```diff
@@ -19,22 +26,41 @@ export enum PermitState {
 
 export interface Permit {
   state: PermitState
+  pending?: boolean
   signature?: PermitSignature
-  callback?: (sPendingApproval: boolean) => Promise<{
+  callback?: () => Promise<{
     response: ContractTransaction
     info: ApproveTransactionInfo
   } | void>
 }
 
 export default function usePermit(amount?: CurrencyAmount<Token>, spender?: string): Permit {
   const { account } = useWeb3React()
-  const tokenAllowance = useTokenAllowance(amount?.currency, account, PERMIT2_ADDRESS)
+  const { amount: tokenAllowance, syncing } = useTokenAllowance(amount?.currency, account, PERMIT2_ADDRESS)
```

Comment:
`isSyncing`

Reply:
I'd like to leave as `syncing` to match the structure of `CallState` (from the multihook library), which is the most relevant prior work with syncable results.